frontend/app/routes/protected/person-case/primary-docs.tsx

```diff
@@ -268,8 +427,132 @@ function DocumentType({ currentStatus, defaultValue, errorMessage }: DocumentTyp
           required
           options={documentTypeOptions}
           label={t('protected:primary-identity-document.document-type.title')}
+          onChange={onChange}
         />
       )}
     </>
   );
 }
+
+interface PrimaryDocsFieldsProps {
+  currentStatus?: string;
+  defaultValues?: {
+    citizenshipDate: Date;
```

Comment:
This should be a string; there's no good reason for it to be a javascript `Date`.